src/ridewithgps_to_cuesheet/conversion.py

```diff
@@ -353,7 +348,7 @@ def _parse_to_cues(array: List[List[str]], opts: GenerationOptions) -> List[Cue]
 
 def _read_as_cue(row: List[str], idx: int, last_dist: Decimal, opts: GenerationOptions) -> Cue:
     has_end = False
-    is_control = row[0] in opts.control_cue_indicators or (row[1].startswith("Control:"))
+    is_control = bool(row[0] in opts.control_cue_indicators or re.match(r"^Control.*?:", row[1]))
```

Comment:
this doesn't need to be a regex does it? startswith is definitely more explicit and we can remove the "colon" if necessary